.gitignore

```diff
@@ -21,6 +21,7 @@ DerivedData
 *.ipa
 *.xcuserstate
 project.xcworkspace
+*.plist
```

Comment:
Hey, this should not be in top level `.gitignore`. See [`Info.plist`](https://github.com/software-mansion/react-native-screens/blob/main/FabricTestExample/ios/FabricTestExample/Info.plist) file which contains application specific configuration we want to have checked into version control.

Reply:
You're right, I can also see there's a couple of `IDEWorkspaceChecks.plist` folks already in the project. Removing this line 👍 https://github.com/software-mansion/react-native-screens/pull/1802/commits/6b23ba464e6cad2b4de6336fffc7f0eb35d6738b